.pre-commit-config.yaml

```diff
@@ -60,25 +60,14 @@ repos:
     rev: 19.10b0
     hooks:
       - id: black
-<<<<<<< HEAD
   # Disabling seed-isort-config in this project (development-guide) since it
   # doesn't identify docopt, et al. as third-party libraries.
   # - repo: https://github.com/asottile/seed-isort-config
-  #   rev: v2.1.0
+  #   rev: v2.1.1
   #   hooks:
   #     - id: seed-isort-config
-  - repo: https://github.com/pre-commit/mirrors-isort
```

Comment:
Why was this removed?  Is it no longer needed?

Reply:
This removal is correct- see https://github.com/cisagov/skeleton-generic/commit/ce6658abc69d1444e51f45c20441c98e6b54c086 and https://github.com/cisagov/skeleton-generic/commit/2399f24622b1d6be92c494653d9308e9694c5496.

Reply:
It was removed upstream: 
https://github.com/cisagov/skeleton-generic/commit/2399f24622b1d6be92c494653d9308e9694c5496#diff-82c3b36709926de3a1b27bea0d8258cc

Paging @mcdonnnj

Reply:
Ah, OK.  My ancient eyes mis-saw what they were looking at.  MY APOLOGIES SIR!!!!!